### ISSUE TITLE
Skip to main should have a focus style

### ISSUE THREAD

**Issue by joshkimux** [AIGAdc/dcdesignweek.org-11ty#99]
![recording](https://netlify-cocoon.netlify.app/.netlify/functions/fetch?code=307&path=eyJzaXRlX2lkIjoiNjk1NmZkOGMtOWNkZS00YmQ1LWE0MTktZjI5ZDY0MGE5OTUyIiwiZGVwbG95X2lkIjoiNjMyZDQ4NDUxOWJiNDEwMDA5NjE4Yzk2IiwiaWQiOiJhMjAxNzFiOC0xMzg5LTRkOTYtYTRiYy0yMzYwMjYxMTBkZTIifQ==)

### Description of issue
Upon tabbing to "skip to main" no focus style is present. Although it's appearing, folks may not realize that it is in fact focused on as there's no visible halo indicating that it is.

### Expected result
Provide a focus halo around the skip to main link.

### Actual result
No focus halo.

### Steps to reproduce
1. Tab once on the homepage 😄 

```
Path:      /
Browser:   Chrome 105.0.0.0 on Mac OS 10.15.7
Viewport:  1792 x 1009 @2x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-62--2022-dcdw.netlify.app%2F&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=105.0&resolution=1792x1009&speed=1&start=true&ref=netlify-source)

Submitted from [Netlify Deploy Preview #62](https://deploy-preview-62--2022-dcdw.netlify.app/)
